### ISSUE TITLE
bigip_pool support description

### ISSUE THREAD

**Issue by johnatspark1** [F5Networks/f5-ansible#55]
##### ISSUE TYPE

 - Feature Idea

##### COMPONENT NAME

bigip_pool
##### ANSIBLE VERSION

`ansible 1.9.4`

```

##### PYTHON VERSION
<!--- Paste verbatim output from “python -V” between quotes below -->
`Python 2.7.9`

```

##### BIGIP VERSION

```

Sys::Version
Main Package
  Product     BIG-IP
  Version     12.1.1
  Build       1.0.196
  Edition     Hotfix HF1
  Date        Wed Sep  7 17:48:09 PDT 2016

Hotfix List
ID593355      ID321971-1  ID589318-1  ID604211-3  ID599068-2  ID594910-1
ID585120-1    ID603997    ID513851-2  ID585366    ID589149-2  ID599054-2
ID601938-2    ID608373-2  ID606110-3  ID590074-1  ID575170-2  ID600778-1
ID598294-1    ID595394-2  ID605682-2  ID590608-1  ID596814-5  ID590578-4
ID601927-1    ID595064-1  ID600859-3  ID599033-3  ID527403-1  ID603605-1
ID614599      ID599769    ID606509-3


```

##### OS / ENVIRONMENT

N/A

##### SUMMARY

Most modules allow us to update the description field.  Not so with the bigip_pool module.  Seems like an easy enhancement.
##### STEPS TO REPRODUCE

```
      - name: add and update pools
        bigip_pool:
            server: "{{ inventory_hostname }}"
            user: "{{ some_username }}"
            password: "{{ some_password }}"
            name: "{{ item.name }}"
            partition: "{{ item.partition }}"
            state: "present"
            **description: "{{ item.description }}"**
        with_items: "{{ poolstable }}"
        when: item.state == "present"

```

##### EXPECTED RESULTS

description of pool to be updated
##### ACTUAL RESULTS

```
TASK: [add and update pools] **************************************************
failed: [10.211.91.33] => (item={'state': 'present', 'partition': 'GEN', 'name': 'testpool', 'description': 'ansible test pool description'}) => {"failed": true, "item": {"description": "ansible test pool description", "name": "testpool", "partition": "GEN", "state": "present"}}
msg: unsupported parameter for module: description

FATAL: all hosts have already failed -- aborting


```

**Comment by caphrim007** [comment]
@johnatspark1 I've added this feature in the bigip_pool code I've just recently pushed. Can you test to ensure that it is working as you expect? If so, I will upstream it to core.

**Comment by johnatspark1** [comment]
It works as expected in my tests.
Well done.  Thanks!

-John Marshall

**Comment by caphrim007** [comment]
@johnatspark1 cool. I'll upstream this then.

**Comment by caphrim007** [comment]
PR for upstream https://github.com/ansible/ansible/pull/19362

**Comment by caphrim007** [comment]
this is now merged upstream and will be part of ansible 2.3
